apache/zeppelin
# [gh-pages][ZEPPELIN-2061] Add Medium blog section to the main page

### What is this PR for?
 There is "[Apache Zeppelin Stories](https://medium.com/apache-zeppelin-stories)" channel in [Medium](https://medium.com/). So I wanted to show the blog posts in this Medium channel via Zeppelin website.  To do this, I used AWS Lambda function: [fetch-zeppelin-medium-story](https://github.com/AhyoungRyu/fetch-zeppelin-medium-story) (read RSS feed -> converted it to JSON) and make it run per every 1 day. So the result file [medium.js](https://s3.amazonaws.com/apache-zeppelin/post/medium.js)(used js file format to easily load to Zeppelin website) will be refreshed everyday. Using this file I added "APACHE ZEPPELIN STORIES" section below the main page of Zeppelin website. 

The reason I did this way to read posts' info is
1. As [Medium/medium-api-docs/issues/49](https://github.com/Medium/medium-api-docs/issues/49#issuecomment-229206837) said, currently there is no API provided by Medium to get the list of blog post info in Story channel ("Apache Zeppelin Stories" is _**Story**_ channel). Instead Medium suggests to [use **RSS feed**](https://help.medium.com/hc/en-us/articles/214874118-RSS-Feeds-of-publications-and-profiles). 

2. and simple :)  

### What type of PR is it?
gh-pages

### What is the Jira issue?
[ZEPPELIN-2061](https://issues.apache.org/jira/browse/ZEPPELIN-2061)

### How should this be tested?
Build `gh-pages` branch as described in [here](https://github.com/apache/zeppelin/tree/gh-pages#build-website). Or see the below gif screenshot img :) 

### Screenshots (if appropriate)
![record](https://cloud.githubusercontent.com/assets/10060731/22653640/60e06a3e-eccd-11e6-8946-9614930541dc.gif)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no
